### ISSUE TITLE
Plugin not working

### ISSUE THREAD

**Issue by videlanicolas** [Wizcorp/phonegap-facebook-plugin#1141]
Hey, I followed the steps, installed the Plugin through Phonegap console, I issue "phonegap plugin ls" and I can see the Facebook Connect plugin installed. Still, I call the following code and nothing happens:

facebookConnectPlugin.login(["public_profile"],
            function(userData){alert("UserInfo: " + JSON.stringify(userData));},
            function (error) { alert("" + error) }
        );

I realised Phonegap is not recognizing the function, but I can see the plugin installed and the folder correctly placed under "plugin". I started a fresh new project just to test this and still the function doesn't execute. Could you give me a hand with this please?

Thanks!

**Comment by pedrokoster** [comment]
Hey, I have a same issue. 
Plugin was installed correctly and I added the following code:
var fbLoginSuccess = function (userData) {
            alert("UserInfo: " + JSON.stringify(userData));
            facebookConnectPlugin.getAccessToken(function(token) {
                alert("Token: " + token);
            }, function(err) {
                alert("Could not get access token: " + err);
            });
        }

        facebookConnectPlugin.login(["public_profile"],
            fbLoginSuccess,
            function (error) { alert("" + error) }
        ); 

Help me.
Thanks

**Comment by videlanicolas** [comment]
I made a fresh Phonegap project, version 0.12.0, installed the plugin via cordova command cli with my app name and ID from Facebook. I pasted my android hash in FB developer and ran the code below:

var options = {
            method: "feed",
            link: "http://example.com",
            caption: "Such caption, very feed."
        };
        facebookConnectPlugin.showDialog(options, function(){alert("Exito!");}, function(){alert("Algo fallo...");});

This is not working. Neither the success function nor the failure function is executing, this usually indicates that the function doesn't exists, But it does exists because I can see it in my plugin folder!

Any halp is welcome, thanks.
